chapters/002-smol-internet/README.md

```diff
@@ -433,38 +358,28 @@ root@tara:/# tcpdump -ne
 18:29:50.570543 02:42:a9:f7:9e:4f > 02:42:0a:01:02:02, ethertype ARP (0x0806), length 42: Reply 10.1.2.1 is-at 02:42:a9:f7:9e:4f, length 28
 ```
 
-From this `tcpdump`, we can see `tara`'s mac address, `02:42:0a:01:02:02`
-attempting to reach `02:42:a9:f7:9e:4f`. But remember, `boudi`'s mac address is
-`02:42:0a:01:01:03`. This tells us that `tara` is making a hail mary to reach
-`10.1.1.3` via the default gateway that docker configured for us automatically.
+From this `tcpdump`, we can see `tara`'s mac address, `02:42:0a:01:02:02` attempting to reach `02:42:a9:f7:9e:4f`. But remember, `boudi`'s mac address is `02:42:0a:01:01:03`. This tells us that `tara` is making a hail mary to reach `10.1.1.3` via the default gateway that docker configured for us automatically.
```

Comment:
Question for another PR... This portion is really about how docker functions, not about how to setup our networks... should we consider removing the bits about docker troubleshooting? 

FWIW, I don't think we can at this point because we don't have the container definitions that we add in our sleep.sh file. But maybe there's a docker solution to defining the network that removes the default gateway?